docs/user-guide/walkthrough/README.md

```diff
@@ -12,77 +12,176 @@ certainly want the docs that go with that version.</h1>
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-# Kubernetes 101 - Walkthrough
+# Kubernetes 101 - Kubectl CLI & Pods
+
+For Kubernetes 101, we will cover kubectl, pods, volumes, and multiple containers
+
+In order for the kubectl usage examples to work, make sure you have an examples directory locally, either from [a release](https://github.com/GoogleCloudPlatform/kubernetes/releases) or [the source](https://github.com/GoogleCloudPlatform/kubernetes).
+
+Table of Contents
```

Comment:
Not yet unfortunately. Replace it with:
```
**Table of Contents**
<!-- BEGIN MUNGE: GENERATED_TOC -->
<!-- END MUNGE: GENERATED_TOC -->
```

and run ./hack/run-gendocs to autogen